.github/workflows/ci_workflows.yml

```
@@ -38,19 +38,17 @@ jobs:
         # Standard tests
         # Linux builds - test on all supported PyQt5/6 and PySide2/6 versions,
         # and include all dependencies in some builds
-        - linux: py38-test-pyqt514-all
-        - linux: py39-test-pyqt515
         - linux: py310-test-pyqt63-all
         - linux: py311-test-pyqt64
         - linux: py312-test-pyqt65
         - linux: py312-test-pyqt68
         - linux: py313-test-pyqt67
         - linux: py311-test-pyqt66-all
         - linux: py311-test-pyqt514
-        - linux: py311-test-pyqt515-lts-all
+        - linux: py314-test-pyqt68
```

```suggestion
        - linux: py314-test-pyqt68-dev
```
Let's try with this one – we haven't actually set up any packages' dev versions (maybe we should?), but this will set the PIP_EXTRA_URL needed to find h5py and shapely downloads – that's what I found for glue-core at least.

---

Should not push these from my phone somewhere in a waiting room... Completely missed the _developer versions_ section below, as well as its glue-core etc. deps! So that should be moved down there of course.